Assets/HoloToolkit-Examples/Input/Scripts/ButtonObjectScaler.cs

```diff
@@ -56,18 +58,14 @@ private void OnButtonPressed(TestButton source)
                         break;
                     case buttonAction.Shrink:
                         if ((ObjectToScale.transform.localScale.x - ScaleIncrement) < 0.0f)
-                        {
-                            ObjectToScale.transform.localScale = new Vector3(0, 0, 0);
-                        }
+                            ObjectToScale.transform.localScale = Vector3.zero;
                         else
-                        {
                             ObjectToScale.transform.localScale = new Vector3((ObjectToScale.transform.localScale.x - ScaleIncrement), (ObjectToScale.transform.localScale.y - ScaleIncrement), (ObjectToScale.transform.localScale.z - ScaleIncrement));
-                        }
```

Comment:
The coding guidelines say that brackets should always be used:
https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/CodingGuidelines.md

This is helpful because if someone later comes along and adds a line, it can be confusing why everything suddenly breaks. Keeping the brackets makes the intention of the code very clear, which is the most important part of code. :)